front/components/items/Appointment.tsx

```diff
@@ -6,35 +6,39 @@ import { DayPicker } from "react-day-picker";
 import { Grid } from "react-loader-spinner";
 
 const Appointment = () => {
-    const therapist = "Bożena"
+    const TEMPtherapist = "Bożena";
+    const filters: filters = {};
     let [chosenDate, setChosenDate] = useState<Date>();
-    let filters: filters = {};
+    let rightTime: Date;
+    let queryChosenDate: string | undefined;
+    useEffect(() => {
+        if (chosenDate) {
+            rightTime = new Date(chosenDate?.getTime() - (-120*60*1000));    
+            // changing timezone by reducing by 2h (because of polish timezone (GMT+2))
+            queryChosenDate = rightTime?.toISOString().slice(0, 10);
+            // changing format of date to YYYY-MM-DD
+            filters.date = {
+                $eq: queryChosenDate
+            };
+        }
+    }, [chosenDate]);
     filters.therapist = {
         first_name: {
-            $eq: therapist
+            $eq: TEMPtherapist
         }
-    }
-    filters.date = {
-        $eq: chosenDate
-    }
+    };
     const [isLoading, setIsLoading] = useState<boolean>(true);
```

Comment:
There is no need to add type boolean when you are starting with initial boolean value. TS should know that type is bool even without those braces `<boolean>`